ietf/secr/meetings/forms.py

```diff
@@ -118,6 +120,9 @@ def save(self, force_insert=False, force_update=False, commit=True):
         meeting.type_id = 'ietf'
         if commit:
             meeting.save()
+            # must call save_m2m() because we saved with commit=False above, see:
+            # https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/#the-save-method
+            self.save_m2m()
         return meeting
```

Comment:
This is trying to keep the magic of the `ModelForm` as much as possible while forcing `type_id='ietf'`. The wrinkle is that `type_id` is a required field, but is in the `exclude` list. If you call `super().save(commit=True)`, an exception will be raised because `type_id` is null.

In order to set `type_id='ietf'`, you need to get a model instance that is otherwise filled in to match the form. With `ModelForm`, saving with `commit=False` does this for you. You can then set `type_id` on the instance and save it.

It's possible there's a less circuitous way to do this, but I didn't see it in a quick search just now (and I feel like I looked for it when I first made this change).

Reply:
Oh - and I don't know why we'd ever want to call this with `commit=False`, but accepting the argument is required to maintain the same API as the superclass.